include/cutlass/gemm/collective/xe_mma.hpp

```diff
@@ -218,20 +198,21 @@ struct CollectiveMma<
     (void)thread_idx;
     (void)smem_buf;
 
+    auto gmem_tiled_copy_a = make_xe_2d_copy(atom_load_A{}.with(mainloop.tensorA),
+                                             Layout<Shape<_1, Int<SubgroupSize>>>{});
+    auto gmem_tiled_copy_b = make_xe_2d_copy(atom_load_B{}.with(mainloop.tensorB),
+                                             Layout<Shape<_1, Int<SubgroupSize>>>{});
+
     // Instantiate the MMA object
     TiledMma tiled_mma;
     auto thread_mma = tiled_mma.get_slice(thread_idx);
-    Tensor tCrA_partition = thread_mma.partition_fragment_A(gA(_, _, 0));
-    Tensor tCrA = make_tensor(static_cast<decltype(tCrA_partition) &&>(tCrA_partition).data(),
-                              tCrA_partition.shape());
-    Tensor tCrB_partition = thread_mma.partition_fragment_B(gB(_, _, 0));
-    Tensor tCrB = make_tensor(static_cast<decltype(tCrB_partition) &&>(tCrB_partition).data(),
-                              make_shape(size<0>(tCrB_partition.shape()),
-                                         size<2>(tCrB_partition.shape()),
-                                         size<1>(tCrB_partition.shape())));
+
+    Tensor tCrA = thread_mma.partition_fragment_A(gA(_, _, 0));
+    Tensor tCrB = thread_mma.partition_fragment_B(gB(_, _, 0));
+
     // Partition the copying of A and B tiles across the threads
-    auto gmem_thr_copy_A = mainloop.gmem_tiled_copy_a.get_slice(thread_idx);
-    auto gmem_thr_copy_B = mainloop.gmem_tiled_copy_b.get_slice(thread_idx);
+    auto gmem_thr_copy_A = gmem_tiled_copy_a.get_slice(thread_idx);
+    auto gmem_thr_copy_B = gmem_tiled_copy_b.get_slice(thread_idx);
 
     auto tCrA_copy_view = gmem_thr_copy_A.retile_D(tCrA);
```

Comment:
Can we do the retile for the gemm, here too? And also a member function of the copy op. If the gemm view is also already retiled here, it doesn't need to be retiled inside the gemm and the gemm doesn't need to know the copy BlockSize.

Even better would be if we can use the same retiled view for both copy and gemm. Why are they different?